tests/specs/pvt/retry-harvesting.e2e.js

```diff
@@ -0,0 +1,33 @@
+import { testTimingEventsRequest } from '../../../tools/testing-server/utils/expect-tests'
+
+describe('pvt harvesting', () => {
+  it('timings are retried when collector returns 429', async () => {
```

Comment:
The retry harvesting for the other features is a bit more robust around different response codes. Is that needed here and/or would that be useful?